modules/appeals_api/spec/support/schema_helpers.rb

```diff
@@ -15,4 +17,58 @@ def read_schema(filename, schema_version = 'v1')
       )
     )
   end
+  # rubocop:disable Layout/LineLength
+
+  def find_max_lengths(filename, _string_to_find, schema_version = 'V2')
+    schema = read_schema(filename, schema_version)
+    max_lengths = {}
+    schema['definitions'].each do |key|
+      if !key[1]['properties'].nil?
+        key[1]['properties'].each do |prop|
+          case key[0]
+          when 'timezone'
+            max_lengths[key[0]] = 'N/A'
+          when 'scCreate'
+            max_lengths['email'] =
```

Comment:
I feel like there should be a more recursive way we could extract the max lengths and their hash paths given _any_ schema. I'll have to mess around with it a bit, but won't block this PR from going through for it :)